nf_core/module-template/modules/main.nf

```diff
@@ -79,8 +79,8 @@ process {{ tool_name_underscore|upper }} {
         $bam
 
     cat <<-END_VERSIONS > versions.yml
-    ${getProcessName(task.process)}:
-        ${getSoftwareName(task.process)}: \$( samtools --version 2>&1 | sed 's/^.*samtools //; s/Using.*\$//' )
+    {{ tool_name_underscore|upper }}:
```

Comment:
Didn't we go for `${task.process}` here?

Reply:
Yeah, forgot to push the change.